packages/contract/scripts/build-contract-call-script.sh

```diff
@@ -2,5 +2,9 @@
 
 set -euo pipefail
 
-forc build -p src/contract-call-script
+SCRIPT_DIR="src/contract-call-script"
+BIN_DIR="$SCRIPT_DIR/out/debug"
+
+forc build -p $SCRIPT_DIR --print-finalized-asm
 pnpm exec ts-node scripts/process-contract-call-script.ts
+# forc parse-bytecode "$BIN_DIR/contract-call-script.bin" > "$BIN_DIR/contract-call-script.txt"
```

Comment:
Remove debugging statement.

Reply:
I didn't forget this, but left it for the next dev to use. I like leaving useful stuff like this around.

Maybe I should put this behind a `FUELS_DEBUG` envvar to make that obvious?